zip-0230.rst

```diff
@@ -0,0 +1,352 @@
+::
+
+  ZIP: 230
+  Title: Version 6 Transaction Format
+  Owners: Daira Emma Hopwood <daira@electriccoin.co>
+          Jack Grigg <jack@electriccoin.co>
+          Kris Nuttycombe <kris@electriccoin.co>
+          Greg Pfeil <greg@electriccoin.co>
+          Deirdre Connolly <deirdre@zfnd.org>
+          Pablo Kogan <pablo@qed-it.com>
+          Vivek Arte <vivek@qed-it.com>
+  Credits: Ying Tong Lai
+  Status: Draft
+  Category: Consensus
+  Created: 2023-04-18
+  License: MIT
+  Discussions-To: <https://github.com/zcash/zips/issues/686>
+
+
+Terminology
+===========
+
+The key words "MUST" and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The character § is used when referring to sections of the Zcash Protocol Specification
+[#protocol]_.
+
+
+Abstract
+========
+
+This proposal defines a new Zcash peer-to-peer transaction format, which includes data that supports Zcash Shielded Assets (ZSAs) on the Orchard shielded pool [#protocol]_.  The
```

Comment:
"data that supports the the Orchard-ZSA protocol and all Zcash Shielded Assets (ZSAs) related operations."

Reply:
Done.